docs/includes/incl-forking-intro.txt

```diff
@@ -0,0 +1,21 @@
+Fork your Aiven service in order to make a copy of the codebase. You can use it to create a development copy of your production, set up a snapshot to analyze an issue or test an upgrade, or create an instance in a different cloud/geographical location/under a different plan 
+
+When you fork the following items are copied into the new service:
```

Comment:
Even though it might add a bit of repetition or mean using "fork" as both a verb and a noun, would it be worth adding something to avoid the standalone "fork" here?

"When you fork a service, the following items..."
OR:
"When you create a fork, the following items..."